src/conda_build_config.rs

```diff
@@ -0,0 +1,223 @@
+//! Module to load deprecated `conda_build_config.yaml` files and apply the selector_config to it
+use std::{collections::BTreeMap, path::Path};
+
+use minijinja::{Environment, Value};
+
+use crate::{
+    selectors::SelectorConfig,
+    variant_config::{VariantConfig, VariantConfigError},
+};
+
+#[derive(Debug)]
+struct ParsedLine {
```

Comment:
I think we could use `ParsedLine<'a>` and use `&'a str` for the Strings. That would safe us from duplicating the contents in memory.